exasol/toolbox/nox/_lint.py

```diff
@@ -49,3 +61,37 @@ def type_check(session: Session) -> None:
     """Runs the type checker on the project"""
     py_files = [f"{file}" for file in python_files(PROJECT_CONFIG.root)]
     _type_check(session, py_files)
+
+
+@nox.session(name="import-lint", python=False)
+def import_lint(session: Session) -> None:
+    """Runs the import linter on the project"""
+
+    usage = "nox -s import-lint -- [options]"
+    description = "Runs the import linter on the project"
+    parser = argparse.ArgumentParser(
+        description=description,
+        usage=usage
+    )
+    parser.add_argument(
+        "-c",
+        "--config",
+        type=str,
+        help="path to the configuration file for the importlinter",
+        metavar="TEXT"
+    )
+
+    args: argparse.Namespace = parser.parse_args(args=session.posargs)
+    file: str = args.config
+    path: Path | None = None
+    if file is None:
+        path = getattr(PROJECT_CONFIG, "importlinter", Path(".importlinter"))
```

Comment:
`import_linter_config` is likely a better name to communicate to the reader of the config file that the setting pertains to the configuration of the tool rather than the tool itself.